pygsti/tools/nameddict.py

```diff
@@ -88,7 +88,7 @@ def _serialize(x):
             #TODO: serialize via _to_memoized_dict once we have a base class
             if x is None or isinstance(x, (float, int, str)):
                 return x
-            elif isinstance(x, _np.int64):
+            elif isinstance(x, (_np.int64, _np.int32)):
```

Comment:
Unrelated serialization fix to address pickle fallback (I think this is a windows specific thing related to the whole 32-bit integer thing that gave us headaches a couple years ago) and a related warning when serializing some RB-related objects.

Reply:
Thanks for catching these sorts of Windows oddities!